src/components/RadioButton/RadioButton.scss

```diff
@@ -1,102 +1,123 @@
-            width: 100%;
-            height: 100%;
-            background: var(--color-brand-50);
-            display: block;
-            transform: scale(0);
-            opacity: 1;
-            border-radius: 50%;
-        }
-
-        &:focus,
-        &:hover {
-            border: 1px solid var(--color-brand-50);
-            stroke: var(--color-highlight-background);
-            background-color: var(--color-highlight-background);
-        }
+        margin-right: var(--space-75);
     }
 
     &__label {
-        margin: auto;
-        user-select: none;
         cursor: pointer;
-        display: block;
 
-        &:focus,
         &:hover {
             .RadioButton__box {
-                border-color: var(--color-brand-50);
+                background: var(--color-background-selected-subtlest-hover, #CCE4FF);
             }
         }
-    }
 
-    &__svg {
-        position: absolute;
-        top: 0;
-        left: 0;
-        fill: var(--color-background);
+        &:active {
+            .RadioButton__box {
+                background: var(--color-background-selected-subtlest-pressed, #99CAFF);
+            }
+        }
     }
 
     &__input {
-        display: none;
+        margin: 0;
+        width: 0;
+        height: 0;
+        border: none;
+        appearance: none;
+
+        &:focus {
+            + .RadioButton__label {
+                .RadioButton__box {
+                    box-shadow: 0px 0px 0px 2px var(--color-info-20, #99CAFF);
+                }
+            }
+        }
 
         &:checked {
             + .RadioButton__label {
                 .RadioButton__box {
-                    background: var(--color-brand-50);
-                    border-color: var(--color-brand-50);
+                    background: var(--color-background-input-default);
+                    border-color: var(--color-background-selected-bold-default);
+                    border-width: 4px;
+                }
+
+                &:hover {
+                    .RadioButton__box {
+                        border-color: var(--color-background-selected-bold-hover, #0062CC);
+                    }
+                }
 
-                    .RadioButton__svg {
-                        stroke-dashoffset: 0;
+                &:active {
+                    .RadioButton__box {
+                        border-color: var(--color-background-selected-bold-pressed, #004999);
                     }
                 }
             }
         }
     }
 
     &__text {
-        padding: var(--space-50) 0 var(--space-50) var(--space-75);
+        composes: OneUI-label-text from global;
+        padding: var(--space-50) 0;
     }
 
     &--disabled {
-        &:hover {
-            .RadioButton__label {
-                .RadioButton__box {
-                    border-color: var(--color-neutral-70);
-                }
-            }
+        /* stylelint-disable no-descending-specificity */
+        .RadioButton__label .RadioButton__box {
```

Comment:
Can we define this style without `.RadioButton__label` part and then remove `/* stylelint-disable no-descending-specificity */`?

Reply:
no, even if you remove it, the issue is still there. I tried for awhile to move things around, but it creates more problem then it solves

Reply:
ok, let's keep as it's now.